ui/src/components/Tabs.vue

```diff
@@ -128,12 +128,12 @@
                 }
             },
             getTabClasses(tab) {
-                const isEnterpriseTab = tab.locked;
+                const isTabLocked = tab.locked;
 
                 return {
-                    "container": !isEnterpriseTab,
-                    "mt-4": !isEnterpriseTab,
-                    "px-0": isEnterpriseTab,
+                    "container": !isTabLocked,
+                    "mt-4": !isTabLocked,
+                    "px-0": isTabLocked,
                 };
```

Comment:
Why create a variable if only use it once

```suggestion
                if(tab.locked){
                    return {}
                };

               return [
                    "container",
                    "mt-4",
                    "px-0",
               ];
```